### ISSUE TITLE
fillExtrusionFloodLight does not light up the ground

### ISSUE THREAD

**Issue by matiasholst** [mapbox/mapbox-maps-android#2176]
## Environment

- Android OS version: API 33
- Devices affected: Tested on Pixel 7 Pro
- Maps SDK Version: 11.0.0-beta.3

## Observed behavior and steps to reproduce

FillExtrusion's FloodLightGroundRadius and other ground related FloodLight features does not seem to have any effect.

![image](https://github.com/mapbox/mapbox-maps-android/assets/15725845/221a77ec-ba31-48c1-9203-a228392c7464)

**Example code**
```kotlin
map.loadStyle(Style.STANDARD)
map.getStyle { style ->

    style.setLight(
        ambientLight {
            intensity(0.005)
        },
        directionalLight {
            intensity(0.005)
        }
    )

    FillExtrusionLayer("fill", "fill").apply {
        fillExtrusionHeight(10.0)
        fillExtrusionColor(Color.GRAY)
        fillExtrusionFloodLightGroundRadius(10.0)
        fillExtrusionFloodLightGroundAttenuation(0.75)
        fillExtrusionFloodLightColor(Color.CYAN)
        fillExtrusionFloodLightWallRadius(5.0)
        fillExtrusionFloodLightIntensity(1.0)
        style.addLayer(this)
    }

    val polygon = Polygon.fromLngLats(
        listOf(
            listOf(
                Point.fromLngLat(-121.58605230635769, 37.73412580656953),
                Point.fromLngLat(-121.5855011124648, 37.73413906437791),
                Point.fromLngLat(-121.58547764313849, 37.73378216334758),
                Point.fromLngLat(-121.5862481075946, 37.733855346759995),
                Point.fromLngLat(-121.58605230635769, 37.73412580656953),

            ),
        )
    )

    style.addSource(geoJsonSource("fill") {
        data(FeatureCollection.fromFeature(Feature.fromGeometry(polygon)).toJson())
    })

}
```

## Expected behavior

Some Cyan light would be cast on the ground.

Something akin to this example which has been made with the JS SDK:

![ground-radius-example](https://github.com/mapbox/mapbox-maps-android/assets/15725845/a505f96c-a207-47e8-a54f-3292aa055b0e)

**Comment by matiasholst** [comment]
I do not have the exact code, but these are the settings that were used for the `FillExtrusion`:
```
 ‘paint’: {
    ‘fill-extrusion-color’: [‘get’, ‘fillColor’],
    ‘fill-extrusion-opacity’: wallOpacity,
    ‘fill-extrusion-height’: [‘get’, ‘height’],
    ‘fill-extrusion-ambient-occlusion-ground-attenuation’: 1,
    ‘fill-extrusion-flood-light-color’: ‘#00FF00’,
    ‘fill-extrusion-flood-light-ground-radius’: 10,
    ‘fill-extrusion-flood-light-intensity’: 0.3,
    ‘fill-extrusion-flood-light-ground-attenuation’: 1,
    ‘fill-extrusion-ambient-occlusion-intensity’: 0
}
```

**Comment by kiryldz** [comment]
Should be fixed in v11.0.0-beta.5 coming out today.
